common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -57,279 +60,175 @@ nav.sequence-nav {
     @include linear-gradient(right, $shadow, $transparent);
     position: absolute;
     top: 0;
-    right: 0;
+    right: ($baseline*2);
     width: 20px;
-    height: 46px;
+    height: 44px;
     background-color: transparent;
     pointer-events: none;
+    z-index: 40;
   }
 
   .sequence-list-wrapper {
     @extend %ui-depth2;
```

Comment:
See my previous comments about ``z-index``. If extending our depth scale from the LMS Sass is possible (as it is here), please use that for my above comments.